app/views/home/businesses.html.erb

```diff
@@ -1,5 +1,5 @@
-<% content_for(:title, "Your business runs on open source software") %>
-<% content_for(:description, "Once a week, every week. Take time to invest in the open source software you use and love.") %>
+<% content_for(:title, t("business_title")) %>
```

Comment:
For content_for() function, key name with space and period does not work on rails I18N.